### ISSUE TITLE
Deployment with Example Data Grid fails

### ISSUE THREAD

**Issue by b-abel** [eclipse/hono#1543]
I tried to deploy a current SNAPSHOT with the Device Connection Service and the example data grid.

I built the images with 

```
# in base directory of Hono working tree:
eval $(minikube docker-env)
mvn clean install -Pbuild-docker-image,metrics-prometheus
```

then tried to deploy it (like in https://www.eclipse.org/hono/docs/dev/deployment/helm-based-deployment/#using-the-device-connection-service) with 

```
cd deploy/target/deploy/helm/
helm install --dep-up --name hono --namespace hono --set deviceConnectionService.enabled=true --set dataGridDeployExample=true eclipse-hono/
```

This failed with the following error message:

```
Error: render error in "eclipse-hono/templates/hono-service-device-connection/hono-service-device-connection-secret.yaml": template: eclipse-hono/templates/hono-service-device-connection/hono-service-device-connection-secret.yaml:47:16: executing "eclipse-hono/templates/hono-service-device-connection/hono-service-device-connection-secret.yaml" at <required "A .Values....>: error calling required: A .Values.deviceConnectionService.hono.deviceConnection.remote needs to be set when deploying the (production) Device Connection service
```

**Comment by sophokles73** [comment]
The second property is not correct. It has been renamed to *dataGridExample.enabled*.

**Comment by sophokles73** [comment]
Should be fixed in https://github.com/eclipse/hono/commit/fd6e2c78b549874634b6fbe6cf806c924620ae5e

@b-abel can you confirm and close this issue?

**Comment by b-abel** [comment]
> Should be fixed in [fd6e2c7](https://github.com/eclipse/hono/commit/fd6e2c78b549874634b6fbe6cf806c924620ae5e)

It is fixed.
